apps/storybook/src/ComboBox.stories.tsx

```diff
@@ -575,3 +577,30 @@ Virtualized.args = {
 Virtualized.argTypes = {
   value: { control: { disable: true } },
 };
+
+export const MultipleSelect: Story<Partial<ComboBoxProps<string>>> = (args) => {
+  const options = React.useMemo(() => countriesList, []);
+  const [selectedOptions, setSelectedOptions] = React.useState(['CA', 'LT']);
+  return (
+    <>
+      <ComboBox
+        options={options}
+        inputProps={{ placeholder: 'Select a country' }}
+        onChange={(value: string, event: string) => {
+          action(value + ' ' + event ?? '')();
+          setSelectedOptions((prev) =>
+            event === 'removed'
+              ? prev.filter((item) => item !== value)
+              : [...prev, value],
+          );
+        }}
+        multiple={true}
+        value={selectedOptions}
+        {...args}
+      />
+      <Button onClick={() => setSelectedOptions(['AL', 'US'])}>
+        change selected
```

Comment:
what's the purpose of this button?

Reply:
To test if combobox works when user changes value. Will be removed for final product.